update deps & switch to github actions

aotarola opened the pull request:
---
_This PR was started by: [git wf pr](https://github.com/groupon/git-workflow/releases/tag/v1.3.1)_

dbushong commented:
This PR doesn't seem to update chromedriver(?)

aotarola commented:
Yeah, I got inspired to switch to gh actions, but forgot to update the title, fixed now 😅

aotarola commented:
@dbushong got a green now, please review 👀

dbushong commented on lib/browser/lighthouse.js:
```diff
@@ -131,18 +131,16 @@ function parseLhResult(results) {
     } else if (scoreDisplayMode === 'notApplicable') {
       notApplicableObj[id] = auditInfo;
     } else if (scoreDisplayMode !== 'manual') {
-      const snippets = [];
```
**nit (non-blocking):** Isn't `snippets` only used inside this else/if clause?  Why not leave the declaration in here?

aotarola replied:
a bad refactoring from my side, fixed!

dbushong reviewed (approved):
LGTM.  What was the reason for the draggable -> pwa switch?  (And score change?)

aotarola commented:
@dbushong, since I updated lighthouse dep; the scoring for PWA, changed as well, so I created a new PWA page test in https://github.com/testiumjs/testium-example-app/pull/21, it should cover it properly now.
